database/generateData.js

```diff
@@ -0,0 +1,25 @@
+
+const fs = require('fs');
+const faker = require('faker');
+
+var options = {
+  autoClose: true
+};
+var writeStream = fs.createWriteStream('seedData.tsv', options);
+var i = 1;
+var photoArr = "{https://loremflickr.com/3000/2000/sanfrancisco,https://loremflickr.com/3000/2000/sanfrancisco,https://loremflickr.com/3000/2000/sanfrancisco,https://loremflickr.com/3000/2000/sanfrancisco,https://loremflickr.com/3000/2000/sanfrancisco,https://loremflickr.com/3000/2000/sanfrancisco,https://loremflickr.com/3000/2000/sanfrancisco,https://loremflickr.com/3000/2000/sanfrancisco,https://loremflickr.com/3000/2000/sanfrancisco,https://loremflickr.com/3000/2000/sanfrancisco}";
+var write = function() {
```

Comment:
I would suggest adding in the number of items you want to create as a parameter. That way this code can be reused to create any number of items, this might also make it a bit easier to test.